docgen/src/guides/prepare-for-v3.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Prepare for v3
+mainTitle: Guides
+layout: main.pug
+category: guides
+withHeadings: true
+navWeight: 0
+editable: true
+githubSource: docgen/src/guides/prepare-for-v3.md
+---
+
+Starting with 2.8.0, we are introducing changes and deprecations to prepare for the upcoming InstantSearch.js 3. InstantSearch has always worked with the [Algolia search client](https://github.com/algolia/algoliasearch-client-javascript) behind the scenes. This will no longer be the default for InstantSearch.js 3. To ease that migration, we recommend applying the changes from this guide now.
```

Comment:
I think we can soften the recommendation :)

> To prepare the future migration to InstantSearch.js v3, you can start using the new options now.

I would still talk about the client only in the introduction of "Initializing InstantSearch".

Reply:
I get it now!